src/Agent.Listener/SelfUpdater.cs

```diff
@@ -9,13 +9,15 @@
 using Microsoft.VisualStudio.Services.Agent.Util;
 using Microsoft.VisualStudio.Services.Common;
 using Microsoft.VisualStudio.Services.WebApi;
+using Newtonsoft.Json;
 using System;
 using System.Diagnostics;
 using System.IO;
 using System.IO.Compression;
 using System.Linq;
 using System.Net.Http;
 using System.Net.Sockets;
+using System.Text.Json;
```

Comment:
Do we have a chance to keep only one Json component, Newtonsoft or System.Text?

Reply:
`System.Text.Json` is not used actually, I removed it